e2e/utils/test.ts

```
@@ -0,0 +1,12 @@
+import { test as base, request as pwRequest } from '@playwright/test';
+import type { APIRequestContext } from '@playwright/test';
+
+export const test = base.extend<{}, { api: APIRequestContext }>({
+  api: [async ({}, use) => {
```

Biome (2.4.4)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)